src/geom_handlers/common_assessment.jl

```diff
@@ -1,6 +1,6 @@
 """ Functions common to both site_assessment methods. """
 
-using LinearAlgebra
+using LinearAlgebra, Dates
```

Comment:
It's good practice to only group "like" packages in import statements

e.g.,

```julia
# These are on separate lines because they are not "like" any of the other packages
using Dates
using FileIO

# This is okay because conceptually they are in the same "group"
using
    Interpolations,
    LinearAlgebra
```

Note that within each "group" of `using`, packages are listed in alphabetical order (this is based on the Blue style guide).

Something else that is often recommended is to import the Base/built-in packages at the top, and more specialist or third-party packages towards the bottom.

Reply:
Same thing goes when exporting functions and methods by the way